docs/getting-started.md

```diff
@@ -154,7 +154,8 @@ generate_study_population: Cleaning up container and volume
 The final line tells you a file of (random) patient data has been created at
 `output/input.csv`, and that it would normally be considered highly sensitive
 data. What you see here reproduces exactly what would happen on a real, secure
-server, but with *dummy data* which contains no real patient information.
+server, but with *dummy data* which contains no real patient information. 
+Remember this is a blank study, read on below to generate *dummy data* values.
```

Comment:
I'm not convinced about this addition - there are quite a few details I've missed out here because I want to keep the word count as short as possible. My hypothesis is that someone who looks at the file and wonders "why is there only one column", will not be confused enough to stop; they'll keep going, and then the next steps make this obvious?

Reply:
i.e. I only think you were confused because you already know what you were looking for